docs/articles/basics.md

```diff
@@ -76,13 +76,17 @@ then it is trivial for someone to gain root access to the host where the
 daemon is running.
 
 Similarly, the Docker client can use `-H` to connect to a custom port.
+The Docker client will default to connecting to `unix:///var/run/docker.sock`
+on Linux, and `tcp://127.0.0.1:2376` on Windows.
 
 `-H` accepts host and port assignment in the following format:
 
-    tcp://[host][:port][path] or unix://path
+    tcp://[host:][port][path] or unix://path
```

Comment:
Wait, am I missing something here?????

What on earth does [path] mean in terms of a daemon listening? At least on Windows, AFAIK, you can only bind/listen/accept in TCP end-point terminology on an IP address/port pair, not a trio of address/port/path. A path is only relevant higher up in the OSI stack such as HTTP. Is there something different on Unix that isn't (to the best of my knowledge) relevant on Windows???

Secondly, IMO the : should remain part of the optional port, not the host. `tcp://1.2.3.4:` seems very strange... That said, I found http://www.tcpipguide.com/free/t_URLGeneralSyntax-2.htm which has the : apparently as it's own entity, so what do I know, lol. I'm sure there's an RFC somewhere which has this expressed 'officially' in BNF, but I couldn't locate it. Perhaps someone with better searching skills would be able to locate it. An RFC would probably be slightly more authoritative than a unit test ;)